app/javascript/components/admin/room_configuration/RoomConfig.jsx

```diff
@@ -29,61 +28,55 @@ export default function RoomConfig() {
-                          subtitle="Prompts the moderator of the BigBlueButton meeting when a user tries to join.
-                            If the user is approved, they will be able to join the meeting."
-                          mutation={() => useUpdateRoomConfig('guestPolicy')}
-                          value={roomConfigs.guestPolicy}
-                        />
-                        <RoomConfigRow
-                          title="Allow any user to start this meeting"
-                          subtitle="Allow any user to start the meeting at any time.
-                            By default, only the room owner can start the meeting."
-                          mutation={() => useUpdateRoomConfig('glAnyoneCanStart')}
-                          value={roomConfigs.glAnyoneCanStart}
-                        />
-                        <RoomConfigRow
-                          title="Allow users join as moderators"
-                          subtitle="Gives all users moderator priviledge in BigBlueButton when they join the meeting"
-                          mutation={() => useUpdateRoomConfig('glAnyoneJoinAsModerator')}
-                          value={roomConfigs.glAnyoneJoinAsModerator}
-                        />
-                        <RoomConfigRow
-                          title="Allow room to be recorded"
-                          subtitle="Allows room owners to specify whether they want the option to record a room or not.
-                            If enabled, the moderator must still click the “Record button once the meeting has started."
-                          mutation={() => useUpdateRoomConfig('record')}
-                          value={roomConfigs.record}
-                        />
-
-                        <RoomConfigRow
-                          title="Viewer Access code"
-                          subtitle="Allows room owners to have a random alphanumeric code that can be shared with users.
-                          The code **if generated** will be required for users to join the room meetings."
-                          mutation={() => useUpdateRoomConfig('glViewerAccessCode')}
-                          value={roomConfigs.glViewerAccessCode}
-                        />
-
-                        <RoomConfigRow
-                          title="Moderator access code"
-                          subtitle="Allows room owners to have a random alphanumeric code that can be shared with users.
-                          The code **if generated** will not be required and when used on any room meeting will join the user as a moderator."
-                          mutation={() => useUpdateRoomConfig('glModeratorAccessCode')}
-                          value={roomConfigs.glModeratorAccessCode}
-                        />
-                      </div>
-                    )
-                  }
+                  <div className="p-4">
+                    <RoomConfigRow
```

Comment:
I see what you're trying to do,
I think that we should figure out why the spinner is glitching and not just omitting its use.
Because this will render the UI but without data, room configs will have no values until the data becomes available. 

PS: For you now you'll see that the configs aren't reflecting their values in the database and that is because of another issue that your PR helped me to spot.
I'll address it in another PR,

Reply:
I put the isLoading / Spinner back in a different way - the glitchy visual is gone but it still lags.
You are right, the isLoading is important.
Maybe we could add the isLoading as a more global so it's not so clumbersome in every component.. just some thoughts.